homeassistant/components/wyoming/satellite.py

```diff
@@ -142,8 +147,8 @@ def _audio_settings_changed(self) -> None:
         # Cancel any running pipeline
         self._audio_queue.put_nowait(None)
 
-    async def _run_once(self) -> None:
-        """Run pipelines until an error occurs."""
+    async def _connect_and_loop(self) -> None:
+        """Connect to satellite and run pipelines until an error occurs."""
         self.device.set_is_active(False)
```

Comment:
Shouldn't this be set to False when you disconnect.

Reply:
That's true. I should probably move that into the except block within `run()`.
This was more of a "getting back into a known state" thing here.